buildelement.js

```diff
@@ -0,0 +1,26 @@
+function build(elementData) {
+  const fs = require('fs');
```

Comment:
You should never require within a function! All requires should be done at the top of the file. This will actually force node to look for this module (perf hit) each time this function is executed.